gradle/utilities.gradle

```
@@ -27,7 +27,7 @@ String runProcess(String... args) {
     def output = process.inputStream.getText()
 
     if (exitCode != 0) {
-        throw new RuntimeException("'${args.join(" ")}' failed with exit code $exitCode. Output was: $output")
+        throw new RuntimeException("\"${args.join(" ")}\" failed with exit code $exitCode. Output was: $output")
```

Is there a reason to change these? The single quotes seem clearer to me.
```suggestion
        throw new RuntimeException("'${args.join(" ")}' failed with exit code $exitCode. Output was: $output")
```

---

I went with consistency (meaning, I used a regex to replace quoting).